content/other/phase-angles/cos/using_this_dataset.md

```diff
@@ -0,0 +1,63 @@
+This dataset contains phase angles to approximate the function $cos(x)$ via [Quantum Signal Processing (QSP)](https://pennylane.ai/qml/demos/function_fitting_qsp) and [Quantum Singular Value Transformation (QSVT)](https://pennylane.ai/qml/demos/tutorial_intro_qsvt).
+
+**Description of the dataset**
+
+QSVT and QSP are powerful quantum algorithms that implement a large class of polynomial transformations. However, many of the operators in these algorithms depend on a series of phase angles that correspond to the desired polynomial. While calculating these angles can be done efficiently in practice, it is not always straightforward. This dataset provides phase angles to implement QSVT and QSP for an approximation of $cos(x)$, making it easy to implement this polynomial without additional calculations.
+
+More specifically, we approximate  $f(x) = cos(2^k2\pi x)$ with a [Chebyshev polynomial](https://en.wikipedia.org/wiki/Chebyshev_polynomials), $P(x)$. The values $k$ and $\epsilon$ define the approximation:
+- **k** is a scaling factor in $f(x)$ and determines the period of the function.
```

Comment:
If I read scaling factor I would assume k*cos(x) or similar. I would say it is a constant